src/core/utils/server.js

```diff
@@ -0,0 +1,63 @@
+/* @flow */
+import path from 'path';
+
+import fs from 'fs-extra';
+import config from 'config';
+import type { $Request, $Response } from 'express';
+
+import log from 'core/logger';
+
+const PREFIX = 'enableFeature';
+
+export const getFeatureFlagName = (featureFlagKey: string): string => {
+  return featureFlagKey.replace(PREFIX, '');
+};
+
+type ExpressHandler = (req: $Request, res: $Response) => void;
+
+type ViewFrontendVersionHandlerParams = {|
+  _config?: typeof config,
+  _log?: typeof log,
+  versionFilename?: string,
+|};
+
+export const viewFrontendVersionHandler = ({
+  _config = config,
+  _log = log,
+  // This is a magic file that gets written by deployment scripts.
+  versionFilename = 'version.json',
+}: ViewFrontendVersionHandlerParams = {}): ExpressHandler => {
+  const version = path.join(_config.get('basePath'), versionFilename);
+
+  const featureFlags = Object.keys(_config)
+    .filter((key) => {
+      return key.startsWith(PREFIX);
+    })
+    .reduce((map, key) => {
+      return {
+        ...map,
+        [getFeatureFlagName(key)]: _config.get(key),
```

Comment:
I still don't see how it's useful to translate the config keys into shorter keys but I don't feel very strong about it. I am worried about confusing someone who would be looking at this output and thinking that it maps to config keys but I guess they would grep for the suffix and find what they're looking for anyway, so 🤷‍♀️